Code/Tools/ProjectManager/tests/SettingsTests.cpp

```diff
@@ -72,105 +86,110 @@ namespace O3DE::ProjectManager
+    TEST_F(SettingsTests, Settings_CopyStringRemoveOriginal_SuccessAndRemovesOriginal)
     {
         QString settingsResult;
-        EXPECT_FALSE(PMSettings::GetProjectManagerKey(settingsResult, m_newSettingsPath));
+        EXPECT_FALSE(m_settings->Get(settingsResult, m_newSettingsPath));
 
         QString settingsValue = "TestValue";
 
         // Don't save to disk in test
-        EXPECT_TRUE(PMSettings::SetProjectManagerKey(m_settingsPath, settingsValue, /*saveToDisk*/ false));
+        EXPECT_TRUE(m_settings->Set(m_settingsPath, settingsValue));
 
-        EXPECT_TRUE(PMSettings::CopyProjectManagerKeyString(m_settingsPath, m_newSettingsPath, /*removeOrig*/ true, /*saveToDisk*/ false));
+        EXPECT_TRUE(m_settings->Copy(m_settingsPath, m_newSettingsPath, /*removeOrig*/ true));
 
         // Check that old path value is removed
-        EXPECT_FALSE(PMSettings::GetProjectManagerKey(settingsResult, m_settingsPath));
+        EXPECT_FALSE(m_settings->Get(settingsResult, m_settingsPath));
 
-        EXPECT_TRUE(PMSettings::GetProjectManagerKey(settingsResult, m_newSettingsPath));
+        EXPECT_TRUE(m_settings->Get(settingsResult, m_newSettingsPath));
         EXPECT_TRUE(settingsResult == settingsValue);
     }
 
-    TEST_F(ProjectManagerSettingsTests, PMSettings_RemoveProjectManagerKey_RemovesKey)
+    TEST_F(SettingsTests, Settings_RemoveProjectManagerKey_RemovesKey)
     {
         QString settingsResult;
-        EXPECT_FALSE(PMSettings::GetProjectManagerKey(settingsResult, m_settingsPath));
+        EXPECT_FALSE(m_settings->Get(settingsResult, m_settingsPath));
 
         QString settingsValue = "TestValue";
 
         // Don't save to disk in test
-        EXPECT_TRUE(PMSettings::SetProjectManagerKey(m_settingsPath, settingsValue, /*saveToDisk*/ false));
-        EXPECT_TRUE(PMSettings::GetProjectManagerKey(settingsResult, m_settingsPath));
+        EXPECT_TRUE(m_settings->Set(m_settingsPath, settingsValue));
+        EXPECT_TRUE(m_settings->Get(settingsResult, m_settingsPath));
 
-        EXPECT_TRUE(PMSettings::RemoveProjectManagerKey(m_settingsPath, /*saveToDisk*/ false));
-        EXPECT_FALSE(PMSettings::GetProjectManagerKey(settingsResult, m_settingsPath));
+        EXPECT_TRUE(m_settings->Remove(m_settingsPath));
+        EXPECT_FALSE(m_settings->Get(settingsResult, m_settingsPath));
     }
 
-    TEST_F(ProjectManagerSettingsTests, PMSettings_GetUnsetBuildPath_ReturnsFalse)
+    TEST_F(SettingsTests, Settings_GetUnsetBuildPath_ReturnsFalse)
     {
         bool buildResult = true;
-        EXPECT_FALSE(PMSettings::GetProjectBuiltSuccessfully(buildResult, m_projectInfo));
+        EXPECT_FALSE(m_settings->GetProjectBuiltSuccessfully(buildResult, m_projectInfo));
         EXPECT_FALSE(buildResult);
     }
 
-    TEST_F(ProjectManagerSettingsTests, PMSettings_SetProjectBuiltSuccessfully_ReturnsTrue)
+    TEST_F(SettingsTests, Settings_SetProjectBuiltSuccessfully_ReturnsTrue)
     {
         // Don't save to disk in test
```

Comment:
you no longer need these `// Don't save to disk in test` comments inside the individual tests

Reply:
Ah good catch